suse2013/common/trim-verbatim.xsl

```diff
@@ -12,61 +12,97 @@
   xmlns:xsl="http://www.w3.org/1999/XSL/Transform" >
 
 <xsl:template match="programlisting/text()|screen/text()|synopsis/text()">
```

Comment:
This could lead to problems. Your XPath checks for **any** text nodes in the matched elements. I think you want to split the above template rule and check for the **first** and the **last** text node explicitly:

```xml
<xsl:template match="programlisting/text()[1]|screen/text()[1]|synopsis/text()[1]"> ... </xsl:template>
<xsl:template match="programlisting/text()[last()]|screen/text()[last()]|synopsis/text()[last()]"> ... </xsl:template>
```